src/main/java/org/broadinstitute/hellbender/tools/genomicsdb/GenomicsDBImport.java

```diff
@@ -140,11 +125,23 @@
                     "to open ~20x as many files. Also, internally GenomicsDB would consume more memory to maintain " +
                     "bookkeeping data from all fragments. Use this flag to merge all fragments into one. " +
                     "Merging can potentially improve read performance, however overall benefit might not be noticeable " +
-                    "as the top Java layers has significantly higher overheads. This flag have no effect if only one " +
+                    "as the top Java layers have significantly higher overheads. This flag has no effect if only one " +
                     "batch is used. Defaults to false",
               optional = true)
     private Boolean doConsolidation = false;
 
+    @Advanced
+    @Argument(fullName = SAMPLE_NAME_MAP_LONG_NAME,
+            shortName = SAMPLE_NAME_MAP_LONG_NAME,
+            doc = "Path to file containing a mapping of sample name to file name in tab delimited format.  If this is " +
+                    "specified then the header from the first sample will be used as the header template rather than " +
```

Comment:
used as the header template -> treated as the merged header rather than actually merging the headers